src/test/java/coverage2/HomeTest.java

```diff
@@ -11,5 +11,11 @@
     public void testHome() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube2.tools.kmb.sonynei.net/coding_rules#rule_key=squid%3AUndocumentedApi)